examples/basic-tracer-node/package.json

```diff
@@ -24,9 +24,9 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.21.0",
```

Comment:
I guess renovate doesn't upgrade example dependencies, since it wasn't included in #2276. Is that intentional?

Reply:
I'm not sure if renovate can be configured to look in the examples also, but it isn't too much of a burden to just do it manually too. I'll look into the renovate config.